apps/openmw/mwworld/containerstore.cpp

```diff
@@ -47,7 +47,7 @@ namespace
         for (typename MWWorld::CellRefList<T>::List::iterator iter (list.mList.begin());
              iter!=list.mList.end(); ++iter)
         {
-            if (Misc::StringUtils::ciEqual(iter->mBase->mId, id2))
+            if (Misc::StringUtils::ciEqual(iter->mBase->mId, id2) && iter->mData.getCount())
```

Comment:
Wow! Major change. I understand why you did put it in there. Let's hope it doesn't break anything else.